scss/tech.scss

```diff
@@ -1,44 +1,54 @@
+.all-cards {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 32px;
+  padding: 40px 40px 40px 100px;
```

Comment:
I recommend you to assign to fixed position, try with `1000px` and later change this for: `margin: 40px auto`